mds/provider_poller/poller.py

```diff
@@ -86,15 +86,16 @@ def _poll_status_changes(self):
 
         # skip
         if polling_cursor == POLLING_CURSOR_TO_QUERY_PARAM.total_events.name:
-            params[polling_cursor] = self.provider.last_skip_polled
-        # recorded
+            p = POLLING_CURSOR_TO_QUERY_PARAM.total_events.value
+            params[p] = self.provider.last_skip_polled
```

Comment:
nit-picking to make it shorter:
```suggestion
            params[POLLING_CURSOR_TO_QUERY_PARAM.total_events.value] = (
                self.provider.last_skip_polled
            )    
```